core/embed/rust/src/strutil.rs

```diff
@@ -9,3 +9,34 @@ pub fn hexlify(data: &[u8], buffer: &mut [u8]) {
         i += 2;
     }
 }
+
+pub fn format_i64(num: i64, buffer: &mut [u8]) -> Option<&str> {
+    let mut i = 0;
```

Comment:
Cannot we theoretically use `inttostr!` instead of this? It is not very practical that calletwe need to supply the buffer every time.

We can call `inttostr!` from anywhere, we would probably need to make it return `Option`/`Result` because it could fail.

Reply:
for now, we want to avoid creating adhoc Strings as much as we can